lib/cli/run-helpers.js

```diff
@@ -81,16 +82,21 @@ exports.list = str =>
  * @returns {Promise<MochaRootHookObject|MochaRootHookFunction>} Any root hooks
  * @private
  */
-exports.handleRequires = async (requires = []) =>
-  requires.reduce((acc, mod) => {
+exports.handleRequires = async (requires = []) => {
+  const acc = [];
+  for (const mod of requires) {
     let modpath = mod;
     // this is relative to cwd
     if (fs.existsSync(mod) || fs.existsSync(`${mod}.js`)) {
       modpath = path.resolve(mod);
```

Comment:
> If the file in question is a package name, path.resolve will fail [...]

Will this work for packages?

Reply:
It should, `fs.existsSync` will return false for a package name, so it is not resolved at all (passed raw) to requireOrImport (e.g. `requireOrImport('@babel/register')`

This behavior is unchanged in this PR